hugo/content/exporter/_index.md

```diff
@@ -163,16 +163,20 @@ CREATE EXTENSION pg_stat_statements;
 | Query File            | Description                                                                                              |
 |-----------------------|----------------------------------------------------------------------------------------------------------|
 | setup.sql    | Creates `ccp_monitoring` role with all necessary grants. Creates all necessary database objects (functions, tables, etc) required for monitoring.  |
+| setup_matview_metrics.sql | Creates materialized views and maintenance objects for them. This feature is optional. See additional info below. |
```

Comment:
instead of 'see below' should we use a footnote marker?

Reply:
Or a section marker, that would work too i think?

Reply:
Did a section marker link